tests/unit/test_client.py

```
@@ -207,6 +207,36 @@ async def test_refresh_auth_success(self, mock_auth, httpx_mock: HTTPXMock):
+        async def fake_update(csrf: str, session_id: str) -> None:
+            calls.append((csrf, session_id))
+            client._core.auth.csrf_token = csrf
+            client._core.auth.session_id = session_id
+
+        monkeypatch.setattr(client._core, "update_auth_tokens", fake_update)
+
+        async with client:
+            refreshed_auth = await client.refresh_auth()
+
+        assert calls == [("new_csrf_token_123", "new_session_id_456")]
+        assert refreshed_auth.csrf_token == "new_csrf_token_123"
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "csrf_token"

(S105)